example/django/README.md

```diff
@@ -1,69 +1,36 @@
 # Bugsnag Django demo
 
-This Django application demonstrates how to use Bugsnag with the Django web
-framework for Python. Before testing it, open up the `bugsnag_demo/settings.py`
-file and configure your API key (see the very bottom of the file).
+This Django application demonstrates how to use Bugsnag with the Django web framework for Python.
 
-```
-BUGSNAG = {
-    "api_key": "066f1ad3590596f9aacd601ea89af845"
-}
-```
+Please note that while bugsnag-python is compatible with previous Python 2 and Django 1 versions -- however, this particular example app will only work with Python 3+ and Django 2+, to showcase the most current configurations. See [our documentation](https://docs.bugsnag.com/platforms/python/django/) for further details.
```

Comment:
I think this is a good idea, but I also think it'd be good to maintain a Django 1.11 example, maybe in a `django_legacy` folder?

Reply:
If anything, I'd call it `django1.11` for clarity

Reply:
sounds good!  I had built one, but then worried it would be too redundant. I'll rename it and submit in a separate PR.

Reply:
added django 1.10 in this same PR.